dealiiadapter.cc

```diff
@@ -378,14 +385,29 @@ class ElasticProblem
     Vector<double> old_displacement;
     Vector<double> displacement;
     Vector<double> old_forces;
+    Vector<double> forces;
     Vector<double> system_rhs;
+    //precice related initializations
+    int             meshID;
```

Comment:
Try to stick to coding style of dealii, so `mesh_id`

Reply:
Maybe there is something similar to the [OpenFOAM Coding Style Guide](https://openfoam.org/dev/coding-style-guide/)? For sure there is a [.clang-format](https://github.com/dealii/dealii/blob/master/.clang-format) file (but I think you use this already).